stdlib/csv/from.go

```diff
@@ -150,16 +170,31 @@ func (c *CSVSource) Run(ctx context.Context) {
 		// transformation. Unlike other sources, tables from csv sources
 		// are not read-only. They contain mutable state and therefore
 		// cannot be shared among goroutines.
-		decoder := csv.NewMultiResultDecoder(csv.ResultDecoderConfig{
+		config := csv.ResultDecoderConfig{
 			Allocator: c.alloc,
 			Context:   ctx,
-		})
+		}
+		switch c.mode {
+		case rawMode:
+			config.NoAnnotations = true
+		default:
+		}
+		decoder := csv.NewMultiResultDecoder(config)
 		var data io.ReadCloser
 		data, err = c.getDataStream()
 		if err != nil {
 			goto FINISH
 		}
-		results, decodeErr := decoder.Decode(data)
+		// Many applications will add a UTF BOM (byte order mark) to the beginning of csv files
+		// We expect UTF8 encoded data so the byte order does not matter.
+		// Therefore we skip the BOM if it exists.
+		// See http://www.unicode.org/faq/utf_bom.html#BOM
```

Comment:
Ah, ok. I did not know about the BOM, or that UTF8 always has the same byte order.